tests/contrib/hooks/test_spark_jdbc_hook.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import unittest
+
+from airflow import configuration, models
+from airflow.utils import db
+
+from airflow.contrib.hooks.spark_jdbc_hook import SparkJDBCHook
```

Comment:
For now these tests are sufficient for me. Maybe in the future we can try to actually perform a spark to postgres job, and assert the result :)

Reply:
That would be a nice test indeed :)

Reply:
Why dont you add one @danielvdende ? Postgres is already available, if you launch a stand-alone spark job it should be possible?

Reply:
This would also require the Spark binary, which is quite heavy, but I really like the idea

Reply:
👍 I'll try to get it working. Just to check I've understood you correctly, you guys mean to use the postgres instance that is started as part of the Travis CI config? (in that case, we could even consider doing the same for MySQL)

Reply:
Try it, but dont over complicate. It’s more an integration test and we should only run those once and not for every setup of Travis. So it might be more for the future. @Fokko can we merge as is?

Reply:
Yes, let me merge it. I've checked it and it looks good to me. 

I really like this operator. This PR introduces a new layer of operators. Instead of having low level operations that just do one thing, we leverage the existing SparkSubmitOperator to add a new layer of logic.